src/utils.ts

```diff
@@ -49,6 +50,25 @@ export function isInsideIframe(): boolean {
   }
 }
 
+/**
+ * Returns true only if application is being iframed inside of Workshop.
```

Comment:
if you use a Workshop in ObjectViews the URL doesn't contain the moduleRid

Reply:
Yes good point...taking a step back it would be a better check here if I went back to isInsideIframe and then checked if the location's pathname started with `/foundry-container-service` which should be consistent across stacks